packages/vm/lib/runBlock.ts

```diff
@@ -84,49 +83,6 @@ export interface RunBlockResult {
   receiptRoot: Buffer
 }
 
-/**
- * Abstract interface with common transaction receipt fields
- */
-interface TxReceipt {
-  /**
-   * Gas used
-   */
-  gasUsed: Buffer
-  /**
-   * Bloom bitvector
-   */
-  bitvector: Buffer
-  /**
-   * Logs emitted
-   */
-  logs: Log[]
-}
-
-/**
- * Pre-Byzantium receipt type with a field
- * for the intermediary state root
- */
-export interface PreByzantiumTxReceipt extends TxReceipt {
-  /**
-   * Intermediary state root
-   */
-  stateRoot: Buffer
-}
-
-/**
- * Receipt type for Byzantium and beyond replacing the intermediary
- * state root field with a status code field (EIP-658)
- */
-export interface PostByzantiumTxReceipt extends TxReceipt {
-  /**
-   * Status of transaction, `1` if successful, `0` if an exception occured
-   */
-  status: 0 | 1
-}
-
-// EIP290Receipt, which has the same fields as PostByzantiumTxReceipt
-export interface EIP2930Receipt extends PostByzantiumTxReceipt {}
```

Comment:
I hate to say this, but moving the receipts from here is likely a breaking change - if people use this importing from `lib/runBlock` then this will break now. So we should likely re-export here and add a deprecation note here and a removal note on the v6 issue?

Reply:
https://github.com/ethereumjs/ethereumjs-monorepo/pull/1185#discussion_r609558696 👍